lite/jupyter_lite_config.json

```diff
@@ -3,6 +3,9 @@
     "output_dir": "./_/_",
     "contents": [
       "../docs"
+    ],
+    "federated_extensions": [
+      "./jupyterlite-pidgy/dist/jupyterlite_pidgy-0.1.0-py3-none-any.whl"
```

Comment:
since we're going with all extensions in `sys.prefix`, we can just do a dev install of this... will want for syntax highlighting anyway

Reply:
not dev installing yet... will do once syntax highlighting...